react/README.md

```diff
@@ -0,0 +1,158 @@
+# Introduction
+
+This React sample demonstrates how to perform login actions that will initiate flow endpoint operations are used only to implement custom authentication UIs. 
```

Comment:
Think this sentence needs to be reworded a bit

Reply:
Working on it

Reply:
Refactored a little. Plus moved it from this repo to the private one in the [same branch](https://github.com/pingidentity/pingone-customers-sample-custom-signon/pull/2/commits/a86c2cc8268d6d220f9b61aa66eb851771095b15)